src/cdi_CPEloss/Analytic_Eloss_Model.hh

```diff
@@ -22,10 +22,10 @@ namespace rtt_cdi_cpeloss {
  * \class Analytic_Eloss_Model
  * \brief Analytic_Eloss_Model base class.
  *
- * This is a base class that defines the interface given to
- * Analytic_Eloss_Model constructors.  The user
- * can define any derived model class that will work with these analytic opacity
- * generation classes as long as it implements the functions required, namely
+ * This is a base class that defines the interface given to Analytic_Eloss_Model
+ * constructors.  The user can define any derived model class that will work
+ * with these analytic opacity generation classes as long as it implements the
+ * functions required, namely.
```

Comment:
Minor detail but could `namely.` -> `namely:` just so subsequent readers don't worry that the comment was accidentally truncated?

Reply:
fixed.